### PR TITLE
chore(portal): add note for multi rg regarding plugins (TDX-2637)

### PR DESCRIPTION
**Pull request by davidma415**
### Summary

Adding a note that some plugins are not compatible if they are not in the default runtime group.

### CODE REVIEW

File: app/konnect/dev-portal/applications/enable-app-reg.md
Comment (Guaris) [inline app/konnect/dev-portal/applications/enable-app-reg.md:138]:
```suggestion
> **Note:** If you are not in the `default` runtime group, consumer-based plugins, like rate-limiting, are not compatible
```

**Comment by Guaris** [comment]
@davidma415 I noticed you added the Do Not Merge label. Is this waiting on other changes?

**Comment by acgoldsmith** [comment]
This is tied to DOCU-2803

**Comment by davidma415** [comment]
> @davidma415 I noticed you added the Do Not Merge label. Is this waiting on other changes? 

I am just awaiting final approval from my team for this! Thanks for the copy edit.

**Review by Mierenga — APPROVED** [review]
LGTM

**Comment by davidma415** [comment]
@Guaris we can merge!

**Comment by acgoldsmith** [comment]
@davidma415 The tech preview badge still needs to be removed.. delete {:.badge .alpha} from line 12.

**Review by acgoldsmith — APPROVED** [review]
LGTM

**Pull request merged**
